layouts/event/single.html

```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
```

Comment:
Please copy the format of the other information under the video. Wrap the license information in a `<div>` and have the label be `<strong>`.